src/components/AddBook.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const categories = ['Fiction', 'Fantasy', 'Action & Adventure', 'Mystery', 'Horror', 'Thriller', 'Biography', 'History', 'Miscellaneous'];
+
+  const dispatch = useDispatch();
+  const [state, setState] = useState({
+    title: '',
+    category: '',
+  });
+
+  const handleChangeName = (e) => {
+    setState((prev) => ({
+      ...prev,
+      [e.target.name]: e.target.value,
+    }));
+  };
+
+  const handleChangecat = (e) => {
+    console.log(e.target.value);
```

Comment:
- [ ] It is not good practice to commit console logs as it can expose confidential data. Kindly remove the console log - it also kills performance.